.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+## 📑 Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## 🍱 Types of changes
+
+What types of changes does your code introduce to AEP?
+_Put an `x` in the boxes that apply_
+
+- [ ] Enhancement
+- [ ] [New proposal](https://aep.dev/1#workflow)
+
+## 📋 Your checklist for this pull request
+Please review the [AEP Style and Guidance](https://aep.dev/style-guide) for contributing to this repository.
+
+### General
+- [ ] Ensure that basic [Guidance](https://aep.dev/style-guide#guidance) is met.
```

Comment:
```suggestion
- [ ] Basic [Guidance](https://aep.dev/style-guide#guidance) is met.
```

nit - just for your consideration. But the "ensure" word doesn't really add a lot of value here - it's a checklist, so by checking it you're implicitly saying you've done what the line has asked you to do.

If you do change it I'd suggest changing it for all lines.